f/dynamicViewDistance/f_setViewDistanceInit.sqf

```diff
@@ -0,0 +1,23 @@
+// FA3 - Dynamic View Distance
+// ====================================================================================
+
+if (!hasInterface) exitWith {};
+
+if (!isDedicated && (isNull player)) then
+{
+	waitUntil {sleep 0.1; !isNull player};
+};
+
+//Set view distance in case the player starts in vehicle
+[player, objNull, vehicle player] spawn f_fnc_ehSetViewDistance;
```

Comment:
Does this work with premount (3den and FA3)? Or does it need to be delayed?

Reply:
In theory it should work since there is no waitUntil/sleep in preMount, so it would run long before the mission is starting.

I just tested this and it seems to work this way.

Reply:
With Eden premount this shouldn't be a problem, since that's defined in the SQM and this stuff runs last via init.sqf.